frontend/src/lib/components/ResizableTable/TableConfig.scss

```diff
@@ -24,13 +23,65 @@
         }
 
         &.selected {
-            background-color: var(--primary-hover);
+            background-color: #eef2ff;
         }
 
         .property-key-info {
             flex-grow: 0;
             overflow: hidden;
             text-overflow: ellipsis;
         }
+
+        .item-label {
+            padding: 0;
+        }
+    }
+
+    .main-content {
+        margin-bottom: 8px;
+
+        .lists {
+            background-color: var(--bg-side);
+            border-radius: 4px;
+            padding: 8px;
+            margin-left: 0 !important;
+            margin-right: 0 !important;
+        }
+    }
+
+    .selected-column-col {
+        display: flex;
+        flex-direction: column;
+    }
+
+    .drag-handle {
+        cursor: move;
+        color: var(--text-default);
+        font-size: 1.2em;
+        padding-right: 5px;
+
+        svg {
+            transform: rotate(90deg) translateX(2px);
+        }
+    }
+
+    .save-as-default-button {
+        border: 1px solid var(--dark-grey);
+        border-radius: 4px;
+    }
+}
+
+.column-configurator-modal-sortable-container {
+    z-index: 9999;
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
+    padding: 0.5rem;
+
+    .drag-handle {
+        color: var(--text-default);
+        font-size: 1.2em;
+        padding-right: 5px;
+        transform: rotate(90deg) translateX(2px);
```

Comment:
Can this be nested inside the other css class?

Reply:
Unfortunately this is a standalone helper class that gets added to the sortable preview during drag, so nesting it won't make it work.

Reply:
I am now investigating how to implement the drag and drop between the two columns.

Reply:
[React Sortable HOC](http://clauderic.github.io/react-sortable-hoc/#/basic-configuration/multiple-lists?_k=pzzyd2) does not support drag and drop between multiple lists, we'd have to implement it ourself, not trivial.